hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/RouterRpcServer.java

```diff
@@ -361,6 +378,23 @@ public RouterRpcServer(Configuration configuration, Router router,
     this.nnProto = new RouterNamenodeProtocol(this);
     this.clientProto = new RouterClientProtocol(conf, this);
     this.routerProto = new RouterUserProtocol(this);
+
+    long dnCacheExpire = conf.getTimeDuration(
```

Comment:
This does not really match the description right? I think you just took it from the other JIRA?

Reply:
This PR is closed, it was by mistake.

I have updated the link to https://github.com/apache/hadoop/pull/2096.